presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoStatement.java

```diff
@@ -194,6 +196,13 @@ public void setCursorName(String name)
         // ignore: positioned modifications not supported
     }
 
+    private Map<String, String> getStatementSessionProperties()
+    {
+        return queryTimeoutSeconds.get() > 0 ?
```

Comment:
This will have to be rewritten if we add more properties. Let's do this using an `ImmutableMap.Builder`